### DISCUSSION TITLE
Where can I find earlier vsix install files of Copilot and Copilot chat

### DISCUSSION THREAD

**Discussion by jimthompson5802** [community/community#81872] [Copilot Conversations]
### Select Topic Area

Question

### Body

Where can I find Copilot Chat vsix files for 0.11.x release?

Reason for asking...
I'm trying to install Copilot and Copilot Chat into a version of code-server that contains vscode 1.85.x.  

Right now I'm able to install Copilot v1.143.601 because it dependency is on vscode ^1.85.0.  

However I am unable to install Copilot Chat 0.12 because it needs vscode ^1.86.0.  

In another Visual Studio Code setup, where I have vscode 1.85.1, I can successfully install Copilot 1.143.601 and Copilot Chat 0.11.1.

I can only seem to find Copilot Chat 0.12.X releases.

**Comment by nerasse** [comment]
Hello 
For me, the best way to solve it is to install "official" microsoft binaries.
I did a repo for it here : https://github.com/nerasse/my-code-server

**Reply by Ram-jannumahanthi** [reply]
this requires root permissions, any options without needing docker/sudo ?

**Reply by nerasse** [reply]
if you have vs code already install you can try this cmd : 
> code serve-web --host 0.0.0.0 --port 8585 --connection-token <token_to_define>

it will serve vscode on port 8585, replace 0.0.0.0 by 127.0.0.1 for local only

**Reply by nerasse** [reply]
replace 0.0.0.0 in the url by your ip or localhost on the host machine if you keep 0.0.0.0 for external network use
